### PR TITLE
Feature/launch jog arm

### LINKED ISSUE #1: [Launch jog_arm] write simulation launch file
* [jog_arm, currently in moveit_experimental](https://github.com/ros-planning/moveit) used for jogging the manipulator
best used together with
  [tork-a/fake_joint](https://github.com/tork-a/fake_joint) if testing jogging in a simulation environment.

### PR DESCRIPTION
**Pull request by Hardthof**
Fixes #1

### CODE REVIEW

File: pilz_jog_arm_support/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- Create a [catkin workspace](http://wiki.ros.org/catkin/Tutorials/create_a_workspace)
+
+## Installation from source
+- clone this package into the src directory `git clone https://github.com/PilzDE/pilz_teach` 
+
+- Install moveit packages from source ...
+  - following the [moveit documentation](https://moveit.ros.org/install/source/)
+  - or with the packages `apt install ros-melodic-moveit` (currently jog_arm is not yet available as binary!)
+
+- install all the dependencies `rosdep install -y --from-paths src --ignore-src`
```
Comment (DamKoVosh) [inline pilz_jog_arm_support/README.md]: (within catkin workspace)

File: pilz_jog_arm_support/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+## Prerequisites
+- You need a workstation or [Virtual machine](https://en.wikipedia.org/wiki/Virtual_machine) with [Ubuntu 18.04 LTS](http://releases.ubuntu.com/releases/18.04/) and [ROS melodic](http://wiki.ros.org/melodic) installed.
+- Create a [catkin workspace](http://wiki.ros.org/catkin/Tutorials/create_a_workspace)
+
+## Installation from source
+- clone this package into the src directory `git clone https://github.com/PilzDE/pilz_teach` 
+
+- Install moveit packages from source ...
+  - following the [moveit documentation](https://moveit.ros.org/install/source/)
+  - or with the packages `apt install ros-melodic-moveit` (currently jog_arm is not yet available as binary!)
```
Comment (DamKoVosh) [inline pilz_jog_arm_support/README.md]: sounds like installing jog_arm from source is optional, maybe extend this line with:
or with the packages ... and only build jog_arm from source

File: pilz_jog_arm_support/launch/prbt_jog_arm_sim.launch [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  <!-- Control a supported gripper, e.g. 'pg70'.
+       Note: Please include the gripper model into your application URDF yourself. -->
+  <arg name="gripper" value="" />
+
+  <include file="$(find prbt_moveit_config)/launch/move_group.launch">
+    <arg name="fake_execution" value="false" />
+    <arg name="load_robot_description" value="true" />
+    <arg name="gripper" value="$(arg gripper)" />
+
+    <arg name="pipeline" value="pilz_command_planner" />
```
Comment (DamKoVosh) [inline pilz_jog_arm_support/launch/prbt_jog_arm_sim.launch:29]: with this pipeline we need depend on pilz_industrial_motion?

Comment (jschleicher) [reply pilz_jog_arm_support/launch/prbt_jog_arm_sim.launch:29]: `pilz_trajectory_generation` is enough. `pilz_industrial_motion` contains the python front-end, while the planner itself lives in `pilz_trajectory_generation`

File: pilz_jog_arm_support/README.md (hunk 2 above)
Comment (jschleicher) [reply pilz_jog_arm_support/README.md]: ah, that was your intention. I'll try to rephrase. I wanted to say "once jog_arm is released, you can simply install the binaries"...

**Review by DamKoVosh — APPROVED** [review]
Tested it on a new linux successfully

**Comment by jschleicher** [comment]
Thanks @DamKoVosh and @Hardthof

**Pull request merged**